src/test/scala/com/gu/subscriptions/cas/directives/ProxyDirectiveSpec.scala

```diff
@@ -76,4 +80,41 @@ class ProxyDirectiveSpec extends FreeSpec
       }
     }
   }
+
+  "proxying to CAS" - {
+    val fooHeader = RawHeader("foo", "bar")
+
+    "proxyRequest" - {
+      val req = HttpRequest(
+        uri = Uri("http://www.example.com/endpoint"),
```

Comment:
Unfortunately the we need to pass a valid host (of course not necessarily a real one) in `Uri.apply`, not just a `"bogus-url"` string